AndroidAnnotations/functional-test-1-5/src/test/java/org/androidannotations/test15/preference/PreferenceEventsHandledActivityTest.java

```diff
@@ -69,6 +69,17 @@ public void testPreferenceChangeParameterPassed() {
 		assertThat(activity.newValue).isSameAs(newValue);
 	}
 
+	@SuppressLint("UseValueOf")
+	@Test
+	public void testPreferenceChangeCastedParameterPassed() {
+		Preference preference = activity.findPreference(activity.getString(R.string.checkBoxWithCastPrefKey));
```

Comment:
The value checking is not needed here, only the preference. We already check the value in the other test.